README.md

```diff
@@ -26,7 +26,34 @@ HADDOCK, standing for **H**igh **A**mbiguity **D**riven protein-protein **DOCK**
 
 ## Installation
 
-Please check the [INSTALL](docs/INSTALL.md) file for instructions.
+Clone the repository and run `pip install`;
+
+```bash
+git clone https://github.com/haddocking/haddock3
+cd haddock3
+pip install .
+```
+
+Execute:
+
+```bash
+$ haddock3 -h
+usage: haddock3 [-h] [--restart RESTART] [--extend-run EXTEND_RUN] [--setup] [--log-level {DEBUG,INFO,WARNING,ERROR,CRITICAL}] [-v] recipe
+
+positional arguments:
+  recipe                The input recipe file path
```

Comment:
```suggestion
usage: haddock3 [-h] [--restart RESTART] [--extend-run EXTEND_RUN] [--setup] [--log-level {DEBUG,INFO,WARNING,ERROR,CRITICAL}] [-v] workflow

positional arguments:
  workflow                The input configuration file path describing the workflow to be performed
```

Reply:
This is coming from the CLI descriptions - will keep it "wrong" just to be consistent

Reply:
Lets modify the CLI then ?
`recipe` is to ambiguous to understand, we are not doing cooking here

Reply:
Modified in #1043